docs/source/contributing.rst

```diff
@@ -179,6 +191,61 @@ A feature development workflow might look like this:
 11. Delete the branch after it has been merged.
 
 
+Adding Data
+-----------
+
+The following steps show how to add new data, ``new_data.npz``, to the packaged data. We assume the SCICO repository has been cloned to ``scico/``.
+
+Note that the data is located in the scico-data submodule, which is
+symlinked to ``scico/data``.  When adding new data, both the scico and
+scico-data repositories must be updated and kept in sync.
+
+
+1. Add the ``new_data.npz`` file to the ``scico/data`` directory.
+
+2. Navigate to the ``data`` directory and add/commit the new data file:
```

Comment:
``scico/data``

Reply:
These instructions have numerous other issues, including the advice to commit directly to the ``main`` branch. Since this goes considerably beyond the intended scope of this PR, I'll create an issue as a reminder that this needs to be addressed.

Reply:
See #152.